ItemDescription3/pageEXT/postedSalesInovicePageExt.50403.PageExt.al

```diff
@@ -0,0 +1,33 @@
+pageextension 50403 apppostedSalesOrderLinePageExt extends "Posted Sales Invoice Subform"
```

Comment:
Please split objects. This is not good practice to have several objects in one file

Reply:
updated